anymail/backends/mailpace.py

```diff
@@ -0,0 +1,190 @@
+                                        message_id=None, status="invalid"
+                                    )
+                                elif "contains a blocked address" in error_message:
+                                    recipient_status[
+                                        email.addr_spec
+                                    ] = AnymailRecipientStatus(
+                                        message_id=None, status="rejected"
+                                    )
+                                elif (
+                                    "number of email addresses exceeds maximum volume"
+                                    in error_message
+                                ):
+                                    recipient_status[
+                                        email.addr_spec
+                                    ] = AnymailRecipientStatus(
+                                        message_id=None, status="invalid"
+                                    )
+                        else:
+                            continue  # No errors found in this field; continue to next field
+            else:
+                raise AnymailRequestsAPIError(
+                    email_message=message,
+                    payload=payload,
+                    response=response,
+                    backend=self,
+                )
+
+        return dict(recipient_status)
+
+
+class MailPacePayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+        }
+        self.server_token = backend.server_token  # esp_extra can override
+        self.to_cc_and_bcc_emails = []
+        self.merge_data = None
+        self.merge_metadata = None
+        super().__init__(message, defaults, backend, headers=headers, *args, **kwargs)
+
+    def get_request_params(self, api_url):
+        params = super().get_request_params(api_url)
+        params["headers"]["MailPace-Server-Token"] = self.server_token
+        return params
+
+    def serialize_data(self):
+        return self.serialize_json(self.data)
+
+    #
+    # Payload construction
+    #
+
+    def init_payload(self):
+        self.data = {}  # becomes json
+
+    def set_from_email(self, email):
+        self.data["from"] = email.address
+
+    def set_recipients(self, recipient_type, emails):
+        assert recipient_type in ["to", "cc", "bcc"]
+        if emails:
+            # Creates to, cc, and bcc in the payload
+            self.data[recipient_type] = ", ".join([email.address for email in emails])
+            self.to_cc_and_bcc_emails += emails
+
+    def set_subject(self, subject):
+        self.data["subject"] = subject
+
+    def set_reply_to(self, emails):
+        if emails:
+            reply_to = ", ".join([email.address for email in emails])
+            self.data["replyto"] = reply_to
+
```

Comment:
I'd suggest implementing `set_extra_headers` to allow *List-Unsubscribe*. Something like:

```python
    def set_extra_headers(self, headers):
        if "list-unsubscribe" in headers:
            self.data["list_unsubscribe"] = headers.pop("list-unsubscribe")
        if headers:
            self.unsupported_features("extra_headers (other than List-Unsubscribe)")
```

(The `headers` param is a CaseInsensitiveDict.)